specification/networkanalytics/NetworkAnalytics.Management/examples/2023-11-15/DataProducts_ListRolesAssignments_MaximumSet_Gen.json

```diff
@@ -0,0 +1,31 @@
+{
+  "title": "DataProducts_ListRolesAssignments_MaximumSet_Gen",
+  "operationId": "DataProducts_ListRolesAssignments",
```

Comment:
>DataProducts_ListRolesAssignments [](http://example.com/codeflow?start=18&length=33)

nit, shouldn't it be 'ListRoleAssignments', not 'ListRolesAssignments'?

Reply:
This is cloned from private repo. We will first make required change in the private repo and make a fix in public repo in the next release.